tools/linux/rewrite_dirs.py

```diff
@@ -4,6 +4,7 @@
 # found in the LICENSE file.
```

Comment:
I don't think we use this any more either.

Reply:
this gets used by `pkg-config-wrapper`, which i believe we still use to wrap around `pkg-config` to generate the correct include and  library paths when cross-compiling using a sysroot

Reply:
I couldn't find any reference to this script, can you link me to where it's used?

Reply:
cc @deepak1556 i think you may know more?

Reply:
This is a pre GN thing, not needed anymore.